src/plugins/discover/public/application/components/context_app/context_app_legacy.tsx

```diff
@@ -73,27 +111,42 @@ export function ContextAppLegacy(renderProps: ContextAppProps) {
       onChangeCount: isPredecessorType ? onChangePredecessorCount : onChangeSuccessorCount,
       isLoading: isPredecessorType ? isLoading(predecessorStatus) : isLoading(successorStatus),
       type,
-      isDisabled: !isLoaded,
+      isDisabled: !isAnchorLoaded,
     } as ActionBarProps;
   };
 
   const docTableProps = () => {
-    const {
-      hits,
-      filter,
-      sorting,
+    return {
+      ariaLabelledBy: 'surDocumentsAriaLabel',
       columns,
+      rows: allRowsLoaded && rows,
```

Comment:
This check needs to prevent first loaded anchor document to be colored. Otherwise see the folowing behaviour. Checkbox from the first row is highlighted 

![9413657A-D110-4E74-B998-4F229FC23510_1_105_c](https://user-images.githubusercontent.com/39378793/117460159-dd4f2f80-af54-11eb-8e67-1fb0811b607a.jpeg)